### DISCUSSION TITLE
Feature request: "Disable device" on GUI

### DISCUSSION THREAD

**Discussion by Isy99** [Koenkk/zigbee2mqtt#12962] [General]
Hello,
would be nice / comfortable to **disable** a device on the Web-Gui per click. I understand "disable" as no more ping, no status update on frontend and mqtt. Device is ignored with other words.
Perhaps an add. "disable device" icon in the same icon line as of "Rename device - Reconfigure - Remove device" (devices tab)

Kind Regards Helmut

**Comment by sjorge** [comment]
Why not just make it leave the network?

**Comment by Isy99** [comment]
Yes that's  always possible, as well you could edit the configuration.yaml as described in the docu. 
A click within the GUI would be nice, nothing more
